WcaOnRails/app/assets/javascripts/associated-events-picker.js

```diff
@@ -3,20 +3,36 @@ $(function() {
     return function() {
       var $eventsFormGroup = $(this).closest('.form-group');
       $eventsFormGroup.find('.event-checkbox input[type="checkbox"]').prop('checked', value);
-      updateEventsSelectedCount($eventsFormGroup);
+      updateEventsInformation($eventsFormGroup);
     };
   }
 
   $('.select-all-events').on('click', checkboxesSetter(true));
   $('.clear-all-events').on('click', checkboxesSetter(false));
 
-  function updateEventsSelectedCount($eventsFormGroup) {
-    var count = $eventsFormGroup.find('input[type="checkbox"]:checked').size();
+  function updateEventsInformation($eventsFormGroup) {
+    var checkedEvents = $eventsFormGroup.find('input[type="checkbox"]:checked');
+    var count = checkedEvents.size();
     var $eventsSelectedCount = $eventsFormGroup.find('.associated-events-label .events-selected-count');
     $eventsSelectedCount.text(count);
+
+    var event_ids = [];
+    for (var i = 0; i < count; i++) {
+      event_ids.push(checkedEvents[i].dataset.event);
+    }
+
+    wca.cancelPendingAjaxAndAjax('render_entry_fee_for_selected_events', {
+      url: 'registrations/event_fee_for_selected_events',
```

Comment:
This url looks wrong to me. Shouldn't it have a competition id in it? I suspect this is working for you because you're visiting a page like https://www.worldcubeassociation.org/competitions/PBQLA2018/register, but what happens if you add a trailing slash to that url and go to something like https://www.worldcubeassociation.org/competitions/PBQLA2018/register/? The page should load, and this relative url you're using will break =(

I think it would make sense for all this logic to go in a javascript function defined in `_register_form.html.erb` instead. That file knows about a specific competition, but I think this file should remain unaware of specific competitions.